src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,41 @@
+import { useNavigate } from 'react-router-dom';
+import React, { useState } from 'react';
+
+import './Header.styles.scss';
+import IconButton from '@components/IconButton';
+import Searchbar from '@components/SearchBar';
+import { ReactComponent as WizelineIcon } from './2D_logo_white.svg';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+function Header(props) {
+  let navigate = useNavigate();
+  let [searchBarFocus, onSearchBarFocusChange] = useState(false);
```

Comment:
These should be `const` since they won't be reassigned.
https://stackoverflow.com/questions/41086633/why-most-of-the-time-should-i-use-const-instead-of-let-in-javascript